public/mobile-app/src/lib/AgendaItem.svelte

```diff
@@ -66,6 +69,9 @@
       padding: 1.5rem;
       padding-bottom: 1rem;
       width: 100%;
+      &.custom {
+        background-image:linear-gradient(0deg,var(--border-default-blue-france),var(--border-default-blue-france)),linear-gradient(0deg,var(--border-default-blue-france),var(--border-default-blue-france)),linear-gradient(0deg,var(--border-default-blue-france),var(--border-default-blue-france)),linear-gradient(0deg,var(--border-default-blue-france),var(--border-default-blue-france))
+      }
```

Comment:
When the 4 gradients are idential, the rule can be simplified by only setting one, which will be used for all 4 sides:

```suggestion
      &.custom {
        background-image: linear-gradient(0deg, var(--border-default-blue-france), var(--border-default-blue-france));
      }
```